diff --git a/dist/client/SuperDebug.svelte b/dist/client/SuperDebug.svelte
index 023ab4b2795c65de31347d60cf42a9f3c807a6fe..14b7c8c2ff2bfffff3389606f32f6ab8f8c17d54 100644
--- a/dist/client/SuperDebug.svelte
+++ b/dist/client/SuperDebug.svelte
@@ -1,4 +1,4 @@
-<script context="module">
+<script module>
 	/*! clipboard-copy. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
 
 	function makeError() {
